diff -ruNp gwenview-4.14.3.orig/CMakeLists.txt gwenview-4.14.3/CMakeLists.txt
--- gwenview-4.14.3.orig/CMakeLists.txt	2014-09-30 11:11:42.000000000 +0200
+++ gwenview-4.14.3/CMakeLists.txt	2019-03-01 13:43:17.190543553 +0100
@@ -32,8 +32,8 @@ macro_log_feature(JPEG_FOUND "libjpeg" "
 find_package(PNG)
 macro_log_feature(PNG_FOUND "libpng" "PNG image manipulation support" "http://www.libpng.org" TRUE)
 
-find_package(Exiv2)
-macro_log_feature(EXIV2_FOUND "Exiv2" "Image metadata support" "http://www.exiv2.org" TRUE "0.19")
+find_package(exiv2)
+macro_log_feature(EXIV2_FOUND "exiv2" "Image metadata support" "http://www.exiv2.org" TRUE "0.19")
 
 macro_optional_find_package(Kipi)
 macro_log_feature(KIPI_FOUND "libkipi" "Provides various image manipulation and export features" "http://www.kipi-plugins.org" FALSE)
