diff -ur gwenview-4.14.3/CMakeLists.txt gwenview-4.14.3-exiv2/CMakeLists.txt
--- gwenview-4.14.3/CMakeLists.txt	2014-09-30 11:11:42.000000000 +0200
+++ gwenview-4.14.3-exiv2/CMakeLists.txt	2019-03-03 13:02:46.118526216 +0100
@@ -32,8 +32,11 @@
 find_package(PNG)
 macro_log_feature(PNG_FOUND "libpng" "PNG image manipulation support" "http://www.libpng.org" TRUE)
 
-find_package(Exiv2)
-macro_log_feature(EXIV2_FOUND "Exiv2" "Image metadata support" "http://www.exiv2.org" TRUE "0.19")
+find_package(exiv2)
+macro_log_feature(exiv2_FOUND "exiv2" "Image metadata support" "http://www.exiv2.org" TRUE "0.19")
+if(exiv2_FOUND)
+    set(EXIV2_LIBRARIES exiv2lib)
+endif()
 
 macro_optional_find_package(Kipi)
 macro_log_feature(KIPI_FOUND "libkipi" "Provides various image manipulation and export features" "http://www.kipi-plugins.org" FALSE)
Only in gwenview-4.14.3-exiv2: CMakeLists.txt~
diff -ur gwenview-4.14.3/lib/exiv2imageloader.h gwenview-4.14.3-exiv2/lib/exiv2imageloader.h
--- gwenview-4.14.3/lib/exiv2imageloader.h	2014-09-30 11:11:42.000000000 +0200
+++ gwenview-4.14.3-exiv2/lib/exiv2imageloader.h	2019-03-03 12:59:49.629697780 +0100
@@ -29,6 +29,7 @@
 
 // Exiv2
 #include <string.h>
+#include <exiv2/exiv2.hpp>
 #include <exiv2/image.hpp>
 
 // Local
